packages/url-loader/src/constants/parameters.ts

```diff
@@ -143,7 +143,23 @@ export const format = {
 
 /** Gravity */
 
-const gravitySchema = z.string();
+const gravitySchema = z.enum([
```

Comment:
From my Googling and AIing, I believe these are all the correct gravity options @colbyfayock.

Reply:
gravity can actually be a wide range of things. iirc it's also an open-ended value where if using the content analysis addon, i believe you can specify anything?

https://cloudinary.com/documentation/transformation_reference#g_gravity

it may be worth adding the ability to pass in a string in addition to the enum so that people have the benefit of the options but also the ability to pass in a custom value, like this:

https://github.com/cloudinary-community/cloudinary-util/blob/main/packages/url-loader/src/constants/parameters.ts#L92